src/classes/frSetupControllerTest.cls

```diff
@@ -5,55 +5,94 @@ public class frSetupControllerTest {
 		Set<String> expectedDonorFieldNames = new Set<String>();
 		Set<String> expectedDonationFieldNames = new Set<String>();
 		for(frField__mdt field : [SELECT DeveloperName, Type__c FROM frField__mdt]) {
-			if(frDonor.TYPE.equals(field.Type__c)) {
+			if('Donor'.equals(field.Type__c)) {
```

Comment:
These changes are not correct and the frDonor/frDonation TYPE variables should be used here.  That was the purpose of having that publicly defined variable that can be shared and managed in one place, so we don't have to update strings in various places

Reply:
Changed!